### PR TITLE
fix: app does not crash on selecting some featured speakers

### LINKED ISSUE #2411: App crashes on selecting some featured speakers
**Actual Behaviour**

App crashes on selecting some featured speakers

**Expected Behaviour**

App should not crash

**Steps to reproduce it**

Open the app. Select featured speakers

**LogCat for the issue**

 `java.lang.IllegalArgumentException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkParameterIsNotNull, parameter string`

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by nikit19**
Fixes #2411 

Changes: app does not crash on selecting some featured speakers

Screenshots for the change: N/A

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/common/utils/Utils.kt [hunk 1]
```diff
@@ -48,10 +48,10 @@ object Utils {
         }
 
         @JvmStatic
-        fun checkStringEmpty(string: String): String {
+        fun checkStringEmpty(string: String?): String {
             var finalString = ""
             if (!isEmpty(string)) {
-                finalString = string
+                finalString = string!!
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/common/utils/Utils.kt]: It's better to just replace this line with `throw new NullPointerException()`

Comment (nikit19) [reply android/app/src/main/java/org/fossasia/openevent/common/utils/Utils.kt]: Okay, I will update the PR

File: android/app/src/main/java/org/fossasia/openevent/common/utils/Utils.kt [hunk 2]
```diff
@@ -48,10 +45,10 @@ object Utils {
         }
 
         @JvmStatic
-        fun checkStringEmpty(string: String): String {
+        fun checkStringEmpty(string: String?): String {
             var finalString = ""
             if (!isEmpty(string)) {
-                finalString = string
+                finalString = string ?: throw  NullPointerException()
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/common/utils/Utils.kt]: This was sarcasm. You obviously don't want to do this. !! does exactly this

Comment (nikit19) [reply android/app/src/main/java/org/fossasia/openevent/common/utils/Utils.kt]: You could have used an emoji to make it look more like a sarcastic comment :laughing: 
sometimes it is difficult to figure out
but I guess I got your point now  :P

File: android/app/src/main/java/org/fossasia/openevent/common/utils/Utils.kt [hunk 3]
```diff
@@ -48,8 +45,8 @@ object Utils {
         }
 
         @JvmStatic
-        fun checkStringEmpty(string: String): String {
-            var finalString = ""
+        fun checkStringEmpty(string: String?): String? {
+            var finalString: String? = ""
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/common/utils/Utils.kt]: Make this function like this

```
if (isEmpty(string))
    return ""
return string
```

Comment (iamareebjamal) [reply android/app/src/main/java/org/fossasia/openevent/common/utils/Utils.kt]: Also change the function name to `nullToEmpty`

**Comment by nikit19** [comment]
Done with the requested changes. Please review

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**